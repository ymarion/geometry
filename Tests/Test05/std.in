# Creating a line
L L1 0 0 20 20
# Should show L1
LIST
#
EXIT
