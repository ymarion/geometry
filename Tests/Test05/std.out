OK
L L1 0 0 20 20


